### PR TITLE
Add rich install for desktop

### PR DESCRIPTION
**Pull request by ramsays**
Desktop now supports a rich install experience with desktop type screenshots. Adding wide screenshots to take advantage of this feature.

https://web.dev/patterns/web-apps/richer-install-ui/

Requesting design review and update to CDN.

### CODE REVIEW

**Comment by vikaspotluri123** [comment]
Can we also use AVIF? I think most if not all browser support it

**Comment by ramsays** [comment]
We can do one or the either as you have to define them one at a time. I chose `webp` so it matches the other ones. `avif` is better compression so I figure we could incorporate that later if do automation on the screenshots.

**Comment by vikaspotluri123** [comment]
Got it, thanks. Do you have an example of how the screenshots look in the installation prompt?

File: lib/templates/webmanifest.js [hunk 1]
```diff
@@ -66,21 +66,42 @@ const renderWebManifest = (name, color) => `
...
+		},
+		{
+			"src": "https://static.gradebook.app/sYbR9JXKTI/screenshot-course-list-wide.webp",
+			"sizes": "1280x720",
+			"type": "image/webp",
+			"form_factor": "wide",
+			"label": "See how you're doing across all your courses - all in one place"
+		},
+		{
+			"src": "https://static.gradebook.app/sYbR9JXKTI/screenshot-course-page-wide.webp",
```
Comment (vikaspotluri123) [inline lib/templates/webmanifest.js:94]: I only see `screenshot-course-wide.webp` in the CDN, was that a typo?

**Review by vikaspotluri123 — CHANGES_REQUESTED** [review]

File: lib/templates/webmanifest.js (hunk 1 above)
Comment (ramsays) [reply lib/templates/webmanifest.js:94]: It was a typo in the file name! I corrected it so it will match the mobile screenshots.

Comment (vikaspotluri123) [reply lib/templates/webmanifest.js:94]: Cool, I dropped `screenshot-course-wide.webp` and added `screenshot-course-page-wide.webp` 👍

**Review by vikaspotluri123 — APPROVED** [review]

**Review by joshcosta — APPROVED** [review]

**Pull request merged**